src/components/features/CardFormFiled/CardNumberForm.tsx

```diff
@@ -0,0 +1,49 @@
+import { css } from '@emotion/react';
+
+import { CardFormFiledProps } from './CardFormFiled.types';
+
+import { CardInputLayout } from '../../common/CardInputLayout';
+import { Flex } from '@/components/common/Flex';
+import { Input } from '@/components/common/Input';
+import { Text } from '@/components/common/Text';
+import { CardInputType } from '@/hooks/useCardInput';
+
+type Props = {
+  cardNumbers: CardInputType[];
+} & CardFormFiledProps;
+
+export const CardNumberForm = ({ cardNumbers, errorMessage, onChange, onBlur }: Props) => {
+  const isInvalidCardNumber = cardNumbers.some((cardNumber) => !cardNumber.isValid);
+
+  return (
+    <CardInputLayout
+      headerText="결제할 카드 번호를 입력해 주세요."
+      description="본인 명의의 카드만 결제 가능합니다."
+      label="카드 번호"
+    >
+      <Flex direction="column" alignItems="flex-start" width="100%" gap="4px">
+        <Flex gap="8px">
+          {cardNumbers.map((cardNumber, index) => (
+            <Input
+              key={`card-${index}`}
+              value={cardNumber.value}
+              onChange={(e) => onChange(index, e)}
+              onBlur={(e) => onBlur(index, e)}
+              isValid={cardNumber.isValid}
+              placeholder="1234"
+            />
+          ))}
+        </Flex>
+        <Text
+          variant="Caption"
+          color="red"
+          css={css`
+            height: 20px;
+          `}
+        >
+          {isInvalidCardNumber ? errorMessage : ''}
+        </Text>
```

Comment:
아래 코드와 어떤 것이 다른가요? 이 방식으로 구현한 이유가 있나요?

```suggestion
        {isInvalidCardNumber ? (
          <Text
            variant="Caption"
            color="red"
            css={css`
              height: 20px;
            `}
          >
            {errorMessage}
          </Text>
        ) : (
          ''
        )}
```

Reply:
에러 메시지가 없으면 해당 위치의 높이가 고정되지 않아서 메시지가 생기면 UI가 조금 틀어지는 것을 막고자 이렇게 구현하게 되었습니다!

혹시 다른 방법이 있다면 힌트를 주실 수 있을까요? 🥲

Reply:
아하 그런 의도였군요. 좋은 것 같습니다!